rewrite-maven/src/main/java/org/openrewrite/maven/search/EffectiveMavenRepositories.java

```diff
@@ -41,6 +47,14 @@ public String getDescription() {
                "determined when the LST was produced.";
     }
 
+    @Option(displayName = "Use markers",
+            description = "Whether to add markers for each effective Maven repository to the POM. Default `false`.",
+            required = false)
+    @Nullable
+    Boolean useMarkers;
+
+    transient EffectiveMavenRepositoriesTable table = new EffectiveMavenRepositoriesTable(this);
```

Comment:
Maybe I would move this as a variable inside the getVisitor method, just to not have transient (because of recipe serde), or at least make it private so it's not exposed.

Reply:
`transient` table seems to be the pattern we follow:

* https://github.com/openrewrite/rewrite/blob/f629bcc3c3f6b9cb6fa696e2a4bed99764c1ebcf/rewrite-core/src/main/java/org/openrewrite/FindCollidingSourceFiles.java#L33
* https://github.com/openrewrite/rewrite/blob/b5f0d7e46441cdd91e82b8600d112366544e4e38/rewrite-core/src/main/java/org/openrewrite/FindParseFailures.java#L47
* https://github.com/openrewrite/rewrite/blob/361b7ba095abaa0381da124049de20355a3a398a/rewrite-core/src/main/java/org/openrewrite/search/FindParseToPrintInequality.java#L32